plugins/module_utils/network/sonic/config/bgp_neighbors/bgp_neighbors.py

```diff
@@ -376,20 +354,6 @@ def remove_default_entries(self, data):
                             new_remote['peer_type'] = peer_type
                     if new_remote:
                         new_neighbor['remote_as'] = new_remote
-                    bfd = neighbor.get('bfd', None)
```

Comment:
If the resulting value of "bfd" is not empty, it needs to be added to "new_neighbor"

Reply:
It's there, you commented on the outdated

Reply:
I see it in the base file. As we discussed, I see that this deleted hunk was a duplicate of the code that remains for handling neighbor BFD configuration deletion.